src/main/scala/io/pinecone/spark/pinecone/package.scala

```diff
@@ -16,7 +16,7 @@ package object pinecone {
       .add("metadata", StringType, nullable = true)
       .add("sparse_values", StructType(
         Array(
-          StructField("indices", ArrayType(IntegerType, containsNull = false), nullable = false),
+          StructField("indices", ArrayType(LongType, containsNull = false), nullable = false),
```

Comment:
Accept Long but reject everything outside of the range of unsgined 32-bit int for sparse indices